src/pages/Dashboard/AddItem/index.tsx

```diff
@@ -0,0 +1,101 @@
+import { FieldValues, useForm } from "react-hook-form";
+import { useTranslation } from "react-i18next";
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as yup from "yup";
+import CustomInput from "../../../components/CustomInput";
+import CustomDatePicker from "../../../components/CustomDatePicker";
+import CustomSelect from "../../../components/CustomSelect";
+import Footer from "./Footer";
+
+/* eslint-disable import/no-anonymous-default-export */
+export default (props: { onClose: Function }) => {
+  const { t } = useTranslation(["dashboard", "error"]);
+  const schema = yup
+    .object({
+      label: yup
+        .string()
+        .required(t("required", { ns: "error" }))
+        .min(
+          4,
+          `${t("min", { ns: "error" })} 4 ${t("characters", { ns: "error" })}`
+        ),
+      description: yup.string(),
+      dueDate: yup.date().required(t("required", { ns: "error" })),
+      priority: yup.string(),
+    })
+    .required();
+  const {
+    register,
+    formState: { errors },
+    trigger,
+    handleSubmit,
+    getValues,
+    setValue,
+  } = useForm({
+    resolver: yupResolver(schema),
+  });
+  const priorities: Array<string> = t("add_item.modal.priority.items", {
+    ns: "dashboard",
+    returnObjects: true,
+  });
+
+  const onSubmit = (values: FieldValues) => {
+    props.onClose();
+  };
+
+  return (
+    <>
+      <CustomInput
+        errors={errors}
+        label="label"
+        register={register}
+        placeholder="Item name"
+        trigger={trigger}
+      />
+      <div className="flex flex-col">
+        <label
+          className={`mb-1 text-sm sm:text-base ${
+            errors["description"] ? "text-red-500" : ""
+          }`}
+          htmlFor="my-input-description"
+        >
+          {t("add_item.modal.description", { ns: "dashboard" })}
+        </label>
+        <textarea
+          className="tracking-wider border-solid border-2 border-slate-300 rounded-lg p-3 text-sm sm:text-base"
+          rows={5}
+          cols={40}
+          {...register("description")}
+          onBlur={() => {
+            console.log(getValues());
```

Comment:
console.log ?
Why do we need console log ?

Reply:
I forgot to remove it